presto-main/src/main/java/io/prestosql/testing/assertions/Assert.java

```diff
@@ -47,4 +49,27 @@ public static void assertEquals(Iterable<?> actual, Iterable<?> expected, String
             throw error;
         }
     }
+
+    public static void assertEventually(Duration timeout, Runnable assertion)
```

Comment:
can we move it to presto-main test jar? or presto-spi test jar?

Reply:
or presto-testing. beyond scope of this PR though